compose/project.py

```diff
@@ -115,6 +117,14 @@ def from_dicts(cls, name, service_dicts, client, use_networking=False, network_d
                     net=net,
                     volumes_from=volumes_from,
                     **service_dict))
+        if config_data.volumes:
+            for vol_name, data in config_data.volumes.items():
+                project.volumes.append(
+                    Volume(
+                        client=client, project=name, name=vol_name,
+                        driver=data.get('driver'), driver_opts=data.get('driver_opts')
+                    )
```

Comment:
Ah ok! I misunderstood. Thank you!

On 14 Dec 2015, at 16:11, Daniel Nephin wrote:

>> @@ -115,6 +117,14 @@ def from_dicts(cls, name, service_dicts, client, 
>> use_networking=False, network_d
>>                   net=net,
>>                   volumes_from=volumes_from,
>>                   **service_dict))
>> +        if config_data.volumes:
>> +            for vol_name, data in config_data.volumes.items():
>> +                project.volumes.append(
>> +                    Volume(
>> +                        client=client, project=name, name=vol_name,
>> +                        driver=data.get('driver'), 
>> driver_opts=data.get('driver_opts')
>> +                    )
>
> @erikh If you want to manage your volumes externally, you'll be able 
> to point at any external volumes as you would normally. This prefixing 
> is only for volumes created locally as part of the compose file.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/docker/compose/pull/2421/files#r47579781